tools/src/main/scala/scala/scalanative/interflow/Inline.scala

```diff
@@ -73,6 +73,8 @@ trait Inline { self: Interflow =>
             alwaysInline || isCtor
           case build.Mode.ReleaseFast =>
             alwaysInline || hintInline || isSmall || isCtor
+          case build.Mode.ReleaseSize =>
+            hintInline || isSmall || isCtor
```

Comment:
I think it should be the other way around, alwaysInline should be respected, hintInline can be ignored.

Reply:
thanks, I've updated, can you please review again?